sdk/include/opentelemetry/sdk/trace/tracer.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "opentelemetry/trace/tracer.h"
+
+namespace opentelemetry
+{
+namespace sdk
+{
+namespace trace
+{
+class Tracer : public opentelemetry::trace::Tracer
+{
+public:
+  explicit Tracer(nostd::string_view name,
+                  nostd::string_view version,
+                  const opentelemetry::trace::TracerOptions &tracer_options = {});
+  nostd::unique_ptr<opentelemetry::trace::Span> StartSpan(
+      nostd::string_view name,
+      const opentelemetry::trace::StartSpanOptions &options = {}) noexcept override;
+
+private:
+  nostd::string_view name_;
+  nostd::string_view version_;
```

Comment:
Don't store string_views like this -- there's no guarantee they'll be point to valid memory through the lifetime of the tracer.

But I don't think the tracer should be passed them anyways.